test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs

```diff
@@ -478,6 +478,37 @@ public void EnvironmentVariablesSettingsShouldSetEnvironmentVariables(RunnerInfo
 
         #endregion
 
+        #region RunSettings defined in project file
+        /// <summary>
+        /// RunSettingsFilePath can be specified in .csproj and should be honored by `dotnet test`, this test 
+        /// checks that the settings were honored by translating an inconlusive test to failed "result", instead of the default "skipped".
+        /// This test depends on Microsoft.TestPlatform.Build\Microsoft.TestPlatform.targets being previously copied into the 
+        /// artifacts/testArtifacts/dotnet folder. This will allow the local copy of dotnet to pickup the VSTest msbuild task.
+        /// </summary>
+        /// <param name="runnerInfo"></param>
+        [TestMethod]
+        [NetFullTargetFrameworkDataSource]
+        [NetCoreTargetFrameworkDataSource]
+        public void RunSettingsAreLoadedFromProject(RunnerInfo runnerInfo)
+        {
+            AcceptanceTestBase.SetTestEnvironment(this.testEnvironment, runnerInfo);
+
+            var projectName = "ProjectFileRunSettingsTestProject.csproj";
+            var projectPath = this.GetProjectFullPath(projectName);
+            this.InvokeDotnetTest(projectPath);
+            this.ValidateSummaryStatus(0, 1, 0);
```

Comment:
consider using named parameters so what these signify is clear.

Reply:
I did consider that, but ended up doing it the same way all the other tests do it to keep it consistent.